### ISSUE TITLE
Problem with a Climate Entity - Wrong Value / Attribute used for Target Temperature

### ISSUE THREAD

**Issue by Tra1n84** [matt8707/ha-fusion#317]
Hej,

I have noticed that an incorrect attribute is being used for the Target temperature at the module for Climate entities:
<img width="373" alt="image" src="https://github.com/matt8707/ha-fusion/assets/72341743/eed89f30-fb77-4f99-bcb4-69bf774e84d8">

The climate module is using the Max. Temperature (see red marking, attribute max_temp) instead of Temperature (see green marking, attribute temperature). See Attributes for this Climate entity from the Developer Tools:
<img width="267" alt="image" src="https://github.com/matt8707/ha-fusion/assets/72341743/d1b3fe7c-8f3d-45ca-ac7d-a46989bdcf9a">

Could you please check this behavior? Or is something wrong on my side?

Thanks & Best regards
Thomas

**Comment by patricknitsch** [comment]
I have the same problem at one from thirteen modules. The other modules work correctly.

**Comment by Tra1n84** [comment]
You're right. I dont have this with other Climate entities either. That's really strange as the other Climate entities tested are of the same Homematic type as the one with the described problem...

With another climate entity of the same model / interface it is correct:
<img width="372" alt="image" src="https://github.com/matt8707/ha-fusion/assets/72341743/b9dbebb3-0ed8-471f-80d8-79eed57618e9">

<img width="260" alt="image" src="https://github.com/matt8707/ha-fusion/assets/72341743/a9c93717-49de-4f56-933c-aad8bebd2bb2">
 
Now the question is why?

**Comment by anarion80** [comment]
I have the same issue for only some climate entities. The issue happens here:
https://github.com/matt8707/ha-fusion/blob/94cd223610a61d3510b303be745f140c27d6d855/src/lib/Components/WheelPicker.svelte#L54

`value` is -1 because my temp is with .1 precision: 12.3, and that doesn't match any calculated temperatures between min and max.

**Comment by mucki12** [comment]
Sorry, but should fix this commit this:
![image](https://github.com/matt8707/ha-fusion/assets/92673795/535b1fa9-af77-49a4-9045-b976225666d6)
![image](https://github.com/matt8707/ha-fusion/assets/92673795/f9a48d8d-f325-47bc-b77d-09f381695a63)
?
